cleanlab_studio/internal/api/api.py

```diff
@@ -917,6 +919,11 @@ async def wrapper(*args: Any, **kwargs: Any) -> Any:
             await asyncio.sleep(sleep_time)
             try:
                 return await func(*args, **kwargs)
+            except ssl.SSLCertVerificationError as e:
+                warnings.warn(
+                    "Please ensure that your SSL certificates are up to date. If you installed python via python pkg installer, please make sure to execute the Install Certificates.command in the python installation directory."
```

Comment:
```suggestion
                    "Please ensure that your SSL certificates are up to date. If you installed python via python pkg installer, please make sure to execute 'Install Certificates.command' in the python installation directory."
```

[very nit] with the weird space in the filename, probably want to make it more clear Install is part of the name in case someone searches for it